homeassistant/components/cloud/client.py

```diff
@@ -136,8 +148,24 @@ async def get_google_config(self) -> google_config.CloudGoogleConfig:
 
         return self._google_config
 
-    async def cloud_started(self) -> None:
-        """When cloud is started."""
+    def _cloud_assist_pipeline(self) -> str | None:
+        """Return the ID of a cloud-enabled assist pipeline or None."""
+        for pipeline in assist_pipeline.async_get_pipelines(self._hass):
+            if (
+                pipeline.conversation_engine == conversation.HOME_ASSISTANT_AGENT
+                and pipeline.stt_engine == DOMAIN
+                and pipeline.tts_engine == DOMAIN
+            ):
+                return pipeline.id
+        return None
+
+    async def create_cloud_assist_pipeline(self) -> None:
+        """Create a cloud-enabled assist pipeline."""
+        await assist_pipeline.async_create_default_pipeline(self._hass, DOMAIN, DOMAIN)
```

Comment:
Why do we need `_cloud_assist_pipeline`, can you' get the pipeline back from this function ?